wire-ios/Wire-iOS/Sources/UserInterface/Settings/Debug Report/ShareableDebugReportView.swift

```diff
@@ -0,0 +1,127 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+import UIKit
+import WireDataModel
+import WireDesign
+
+class ShareableDebugReportView: UIView {
+
+    // MARK: - Views
+
+    private let topLabel: UILabel = {
+        let topLabel = UILabel()
+        topLabel.numberOfLines = 1
+        topLabel.lineBreakMode = .byTruncatingMiddle
+        return topLabel
+    }()
+
+    private let bottomLabel: UILabel = {
+        let bottomLabel = UILabel()
+        bottomLabel.numberOfLines = 1
+        return bottomLabel
+    }()
+
+    private let labelsView = UIView()
+
+    private let documentIconView: UIImageView = {
+        let documentIconView = UIImageView()
+        documentIconView.tintColor = SemanticColors.Icon.backgroundDefault
+        documentIconView.contentMode = .center
+        documentIconView.setTemplateIcon(.document, size: .small)
+        return documentIconView
```

Comment:
We want to move away from StyleKit. I would recommend checking the Figma file, export the icon directly from there as SVG. And then use it in the Xcode Project.

Reply:
done https://github.com/wireapp/wire-ios/pull/1658/commits/8c85d9ddbe3d4f96ae06fb91a9db48865e963819